packages/prosemirror-lwdita/src/github-integration/github.plugin.ts

```diff
@@ -113,13 +113,13 @@ export const getUserInfo = async (token: string): Promise<Record<string, string>
  * @param changedDocument - The content of the changed document.
  * @returns A promise that resolves when the document has been published.
  */
-export const createPrFromContribution = async (ghrepo: string, source: string, changedDocument: string, title: string, desc: string): Promise<string> => {
+export const createPrFromContribution = async (ghrepo: string, source: string, branch: string, changedDocument: string, title: string, desc: string): Promise<string> => {
   const authenticatedUserInfo = await getUserInfo(localStorage.getItem('token') as string);
   
   const owner = ghrepo.split('/')[0];
   const repo = ghrepo.split('/')[1];
   const newOwner = authenticatedUserInfo.login;
-  const newBranch = "new-branch-petal-app";
+  const newBranch = "new-branch-petal";
```

Comment:
What is this `new-branch-petal` used for?

Reply:
This is a new branch created by the GitHub App to push changes to, and create the PR from.

Reply:
Okay interesting. Then I think we need to improve on this. Especially as they might want to use Petal to edit more than one page in the same repo, and if we hardcode the branch name, we will get a conflict when they edit a second page. Perhaps this should be something like `doc/petal-${timestamp}` where the timestamp is a serial timestamp like `YYYYMMDDHHmmSS` (on the UTC timezone)

Reply:
Fixed in 58516fd2a46ddf0292097ac7cfa6e33bb293725d